packages/focusjail/src/FocusJailContainer.spec.tsx

```diff
@@ -83,11 +93,14 @@ describe('FocusJailContainer', () => {
 
       expect(() => {
         render(
+          /* eslint-disable @typescript-eslint/ban-ts-ignore */
+          // @ts-ignore
```

Comment:
`@ts-ignore` is used here because this test is verifying that a runtime error is thrown when the required argument, `containerRef` isn't passed. In this test case, it's okay that `containerRef` isn't passed.